kommunicate/src/main/java/io/kommunicate/Kommunicate.java

```diff
@@ -306,7 +306,7 @@ public void onFailure(Object error) {
     private static void createConversation(KmChatBuilder chatBuilder, KMStartChatHandler handler) throws KmException {
         List<KMGroupInfo.GroupUser> users = new ArrayList<>();
 
-        KMGroupInfo channelInfo = new KMGroupInfo(TextUtils.isEmpty(chatBuilder.getChatName()) ? "Kommunicate Support" : chatBuilder.getChatName(), new ArrayList<String>());
+        KMGroupInfo channelInfo = new KMGroupInfo(TextUtils.isEmpty(chatBuilder.getChatName()) ? "Support" : chatBuilder.getChatName(), new ArrayList<String>());
```

Comment:
is this the default name? we can make it same as web, i guess it is "Conversations" on web. @Rudranand pls confirm

Reply:
done